.github/workflows/jekyll.yml

```
@@ -0,0 +1,34 @@
+name: CI for GitHub Pages site
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    # Ruby-specific steps
+    - name: Set up Ruby 2.6
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.6.x
+    - name: Build and test with Rake
+      run: |
+        gem install bundler
+        bundle install --jobs 4 --retry 3
+        bundle exec rake
+
+    # Node-specific steps
+    - name: Install node modules
+      run: |
+        npm install
+
+    # Jekyll / GitHub-pages specific steps
+    - name: Build the site in the jekyll/builder container
+      run: |
+        docker run \
+        -v ${{ github.workspace }}:/srv/jekyll -v ${{ github.workspace }}/_site:/srv/jekyll/_site \
+        jekyll/builder:latest /bin/bash -c "chmod 777 /srv/jekyll && jekyll build --future"
```

@mattdavis0351 I think this is just running a `jekyll build`, which is the command that'll tell us if there are any build errors, within a container. I don't think it's using the container to deploy anywhere.

---

```suggestion
        jekyll/builder:4 /bin/bash -c "chmod 777 /srv/jekyll && jekyll build --future"
```

When we use the tag `latest` for a Docker image we pull the most current version that is uploaded to the registry.  That's usually fine as we develop things, but when we want there to be some sort of long term life to this workflow it would benefit us to be explicit about the version of the image we are using.  In this case `latest` is at version `4` so we can replace that tag appropriately and we will know for certainty that every time this workflow runs it pulls the same image, thus making sure that nothing breaks in six months.

---

If you think it's a good idea we can do that, @mattdavis0351, but I should mention that I'm using the [templated workflow](https://github.com/actions/starter-workflows/blob/master/ci/jekyll.yml) for Jekyll from GitHub.